test/Microsoft.Dnx.Testing.CommonUtils/DnxSdk/DnxSdk.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Runtime.Versioning;
+
+namespace Microsoft.Dnx.Testing
+{
+    public class DnxSdk
+    {
+        public string Version { get; set; }
+
+        public string Flavor { get; set; }
+
+        public string Architecture { get; set; }
+
+        public string OperationSystem { get; set; }
+
+        public string Path { get; set; }
+
+        public string FullName { get; set; }
+
+        public FrameworkName TargetFramework { get; set; }
+
+        public Dnu Dnu => new Dnu(Path);
+
+        public Dnx Dnx => new Dnx(Path);
+
+        public static string GetRuntimeHome()
+        {
+            var homePath = Environment.ExpandEnvironmentVariables(Environment.GetEnvironmentVariable("DNX_HOME"));
```

Comment:
I didn't find a suitable helper in the current code base. There is something in DnuEnvironment.cs but that seems to be ifdef'ed out for only DNXCORE50 so I'm not sure if that's a good idea.